test/utils/secret.go

```diff
@@ -0,0 +1,34 @@
+package utils
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+type Secret struct {
+	TestManualConnection bool
+}
+
+func TestSecret(t *testing.T, consumerName string, conf Secret) {
+	t.Run("secrets interface", func(t *testing.T) {
+		if conf.TestManualConnection {
+			SnapConnect(nil,
+				"edgexfoundry:edgex-secretstore-token",
+				"edgex-"+consumerName+":edgex-secretstore-token",
+			)
+		}
+		requireCorrectSecret(t, consumerName)
```

Comment:
I suggest moving this to main, after installing the snaps.

Reply:
I have moved it to main, under the SnapInstallFromFile section, because the Snap connection is only needed when installing from a file. Does this sound good?

Reply:
Yes. Looks cleaner.